src/solver/main.cpp

```diff
@@ -200,8 +194,8 @@ bool SolverApplication::prepare(int argc, char* argv[])
                   << it->second
                   << ". Conflict between Min Stable Power, Pnom, spinning and capacity modulation.";
             }
-
-            return false;
+            // TODO : create & throw exception
```

Comment:
I see, what's preventing you from throwing an exception here is that there a loop. It should be a problem though, and it would allow this function to return **void**

Reply:
done